presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigUtils.java

```diff
@@ -35,14 +35,20 @@
 
 public class PrestoConfigUtils
 {
-    private static final Path DEFAULT_CONFIG_DIR = Paths.get("/presto-manager/config");
-    private static final Path DEFAULT_CATALOG_DIR = Paths.get("/presto-manager/config/catalog");
-    private static final Path DATA_DIR = Paths.get("/var/lib/presto/data");
-    private static final Path PLUGIN_DIR = Paths.get("/usr/lib/presto/lib/plugin");
+    private final Path defaultConfig;
+    private final Path defaultCatalog;
+
+    private CommandExecutor executor;
 
-    private PrestoConfigUtils() {}
+    @Inject
+    PrestoConfigUtils(AgentConfig config, CommandExecutor executor)
```

Comment:
It's weird for a utils class to be instantiable. If it's doing a job, maybe PrestoConfigurationManager? If it's a bundle of methods, maybe it could merge with some other class that does stuff on the cluster.